nnvm/python/nnvm/compiler/build_module.py

```diff
@@ -247,6 +254,36 @@ def build(graph, target=None, shape=None, dtype="float32",
 
     cfg = BuildConfig.current
     graph = graph if isinstance(graph, _graph.Graph) else _graph.create(graph)
+    # Build extra operator runtime library
+    extra_lib = ()
+    build_extra = False
+    if cfg.extra_lib_op is not None:
+        build_extra = True
+        graph, extra_op_graph = graph_util.split_last_op(graph)
+        last_op_name = extra_op_graph.index.nodes[-1]["op"]
+        if cfg.extra_lib_op != last_op_name:
+            raise RuntimeError("Currently only supports splitting the "
+                               "last operator of the input graph. "
+                               "extra_lib_op in build_config is %s, "
+                               "but the last op of the graph is %s." %
+                               (cfg.extra_lib_op, last_op_name))
+        extra_op_params = {}
+        if params is not None:
+            for input_name in extra_op_graph.symbol.list_input_names():
+                if input_name in params:
+                    extra_op_params[input_name] = params[input_name]
+                    params.remove(input_name)
```

Comment:
extra_lib op can be used generically.
Removing params !! How about param is being shared ??

A bit unrealistic, but making it to be aware.